src/Create.js

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import { useHistory } from "react-router-dom";
+
+const Create = () => {
+    const [title, setTitle] = useState('');
+    const [body, setBody] = useState('');
+    const [author, setAuthor] = useState('mario');
+    const [isPending, setIsPending] = useState(false);
+    const history = useHistory();
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        const blog = { title, body, author };
+
+        setIsPending(true);
+
+        fetch('http://localhost:8000/blogs', {
```

Comment:
Use single quotes or double quotes. Do not mixing both without any  specific reason.